akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -274,16 +276,19 @@ shard resolution, e.g. to avoid too fine grained shards. Once a shard's location
 the only overhead is sending a message via the `ShardRegion` rather than directly.
 
 <a id="cluster-sharding-mode"></a>
-## Distributed Data vs. Persistence Mode
+## Sharding State Store Modes
 
-The state of the coordinator and the state of [Remembering Entities](#cluster-sharding-remembering) of the shards
-are persistent (durable) to survive failures. @ref:[Distributed Data](distributed-data.md) or @ref:[Persistence](persistence.md)
-can be used for the storage. Distributed Data is used by default.
+There are two cluster sharding states managed:
+1. @ref:[ShardCoordinator State](#shardcoordinator-state) - the `Shard` locations
+1. @ref:[Remembering Entities](#remembering-entities) - the entities in each `Shard`, which is optional, and disabled by default
+ 
+For these, there are currently two modes which define how these states are stored:
+* @ref:[Distributed Data Mode](#distributed-data-mode) - uses Akka @ref:[Distributed Data](distributed-data.md) (CRDTs) (the default)
+* @ref:[Persistence Mode](#persistence-mode) - uses Akka @ref:[Persistence](persistence.md) (Event Sourcing)
```

Comment:
"which is deprecated and will be removed"?

Reply:
Persistence, if you clicked on the `#persistence-mode` link, it's noted there.